commands/apps.go

```diff
@@ -225,6 +225,32 @@ Only basic information is included with the text output format. For complete app
 	)
 	AddStringFlag(updateAlertDestinations, doctl.ArgAppAlertDestinations, "", "", "Path to an alert destinations file in JSON or YAML format.")
 
+	CmdBuilder(
+		cmd,
+		RunAppListBuildpacks,
+		"list-buildpacks",
+		"List buildpacks",
+		`List all buildpacks available on App Platform`,
+		Writer,
+		displayerType(&displayers.Buildpacks{}),
+	)
+
+	upgradeBuildpack := CmdBuilder(
+		cmd,
+		RunAppUpgradeBuildpack,
+		"upgrade-buildpack <app id>",
+		"Upgrade buildpack",
+		`Upgrade a buildpack for an app`,
+		Writer,
+		displayerType(&displayers.Deployments{}),
+	)
+	AddStringFlag(upgradeBuildpack,
+		doctl.ArgBuildpack, "", "", "The ID of the buildpack to upgrade", requiredOpt())
```

Comment:
nit: Might be worth calling out in help messaging they can list the existing versions with the `list-buildpacks` command.

Reply:
good idea